### ISSUE TITLE
Allow using any node attribute

### ISSUE THREAD

**Issue by rafguns** [rafguns/nx2vos#1]
We currently expect in `write_vos_map` that the name and format of `attrs` already follows the VOSviewer conventions. For instance, we expect an attribute `cluster` to have a numeric value between 1 and 1000. It would be more user-friendly if we could convert to this format from, e.g., node attribute 'department', which has string values.

**Comment by rafguns** [comment]
API could look like this:

```python
def write_vos_map(
    G: nx.Graph,
    fname: str | pathlib.Path,
    *,
    sublabel_attr: str | None = None,
    description_attr: str | None = None,
    url_attr: str | None = None,
    x_attr: str | None = None,
    y_attr: str | None = None,
    cluster_attr: str | None = None,
    weight_attrs: list[str] | None = None,
    score_attrs: list[str] | None = None,
):
    ...
```

It's a lot of parameters but I expect one would typically need just a few of them. I don't think we should support 'normalized weight', 'red', 'green', and 'blue'.

**Comment by rafguns** [comment]
Or maybe better:

```python
def write_vos_map(
    G: nx.Graph,
    fname: str | pathlib.Path,
    attribute_map: dict[str, str] | None = None,
):
    ...
```

which could be called like this `write_vos_map(G, fname, {"department": "cluster", "pubcount": "weight", "citcount": "weight"})`.

**Comment by rafguns** [comment]
Fixed by #2
